lib/reporters/fps.js

```diff
@@ -1,9 +1,10 @@
 const babar = require('babar');
 
 function chart (data, url) {
-  return babar(data.map((fps, i) => {
+  var series = data.map((fps, i) => {
     return [i, fps];
-  }), { caption: `scrolling fps - ${url}`, height: 15 });
+  })
+  return babar(series, { caption: `scrolling fps - ${url}`, height: 12, maxX: 20, minY: 1, maxY: 60 });
```

Comment:
I don't think a hard coded `maxX` value is a good idea as it will prevent long running fps experiments, and (although I'm not certain of this) make the graph misleading for experiments that run less than 20s.